pkgs/games/crossfire/crossfire-arch.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchsvn, version, rev, sha256 }:
+
+stdenv.mkDerivation rec {
+  version = "r${toString rev}";
+  name = "crossfire-arch-${version}";
+
+  src = fetchsvn {
+    url = "http://svn.code.sf.net/p/crossfire/code/arch/trunk/";
+    sha256 = sha256;
+    rev = rev;
+  };
+
+  phases = ["unpackPhase" "installPhase"];
+  installPhase = ''
+    mkdir -p "$out"
```

Comment:
This is a no-op. You can just add a meta attribute to `fetchsvn` and return it instead.

Reply:
I'm not sure you mean by this.

Reply:
Right now you are building a derivation that returns the content of the resource attribute.
This creates an identical derivation with identical content of the `fetchsvn` derivation. 
Instead you can just say:

```
fetchsvn {
    url = "http://svn.code.sf.net/p/crossfire/code/arch/trunk/";
    sha256 = sha256;
    rev = rev;
    meta = with lib; {
      description = "Archetype data for the Crossfire free MMORPG";
      homepage = "http://crossfire.real-time.com/";
      license = licenses.gpl2;
      platforms = platforms.linux;
      maintainers = with maintainers; [ ToxicFrog ];
  };
};
```

instead of 
```
stdenv.mkDerivation {
   ...
   src = ...;
}
```